docs/xbcloud-binary-fifo-datasink.md

```diff
@@ -0,0 +1,58 @@
+# FIFO datasink
+
+[Percona XtraBackup 8.0.33-28](release-notes/8.0/8.0.33-28.0.md) implements a new datasink that uses `FIFO` (named pipes). With FIFO datasink feature, you can benefit from faster backups by streaming data in parallel from XtraBackup to `xbcloud`.
+
+## Stream to object storage
+
+When taking a backup, you can save the files locally or stream the files to either different servers or object storage. 
+
+Streaming backups to object storage using [`xbcloud`](xbcloud-binary-overview.md) works as follows:
+
+* XtraBackup spawns multiple copy threads. 
+
+* Each copy thread reads a chunk of data from a specific file.
+
+* Each copy thread writes the chunk of data to a pipe (STDOUT).
+
+* Xbcloud has a read thread that is responsible for reading each chunk of data from STDIN (standard input device). This chunk is uploaded to an object storage using an async request, and a callback is added to an event handler list.
```

Comment:
A read thread reads each chunk of STDIN (standard input device) data. xbcloud uploads the chunks to an object storage ...

Reply:
done